.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             - workspace
 
       - save_cache:
-          key: trace-java-{{ checksum "trace-java.gradle" }}-{{ .Branch }}-{{ .Revision }}
```

Comment:
Since you still have the version number set in `trace-java.gradle`, you probably don't need to make this change.  We rely purely on tags now for releases, so that file doesn't change which is why we have to be more careful.

Reply:
Oh, good to know. @iNikem happens to be removing the version number in `trace-java.gradle` right now in #403.